jobs/kpi-forecasting/README.md

```diff
@@ -65,14 +64,34 @@ cd ~/PATH/TO/CONDA/envs/kpi-forecasting-dev/lib/python3.10/site-packages/prophet
 install_name_tool -add_rpath /PATH/TO/CONDA/envs/kpi-forecasting-dev/lib/cmdstan-2.32.2/stan/lib/stan_math/lib/tbb prophet_model.bin
 ```
 
+### Running locally
+A metric can be forecasted by using a command line argument that passes the relevant YAML file to the `kpi_forecasting.py` script.
+[Here are approaches for accessing a Docker container's terminal](https://docs.docker.com/desktop/use-desktop/container/#integrated-terminal).
+
+For example, the following command forecasts Desktop DAU numbers:
+
+```sh
+python ~/kpi_forecasting.py -c ~/kpi_forecasting/configs/dau_desktop.yaml
+```
+
+Similarly, the model performance for the KPI forecast can be done by executing:
+```sh
+python ~/model_performance_analysis.py.py -c ~/kpi_forecasting/configs/kpi_model_validation.yaml   
```

Comment:
I left it as `performance_analysis` because it is also used for the search forecasting.

Yeah the README is kind of stale.  I did make sure the commands relevant to the PR worked but I'd say we check and update the rest of it in a separate PR.